.cbmc-batch/include/openssl/evp.h

```diff
@@ -45,6 +51,13 @@ int EVP_PKEY_CTX_set_rsa_mgf1_md(EVP_PKEY_CTX *ctx, const EVP_MD *md);
 int EVP_PKEY_encrypt(EVP_PKEY_CTX *ctx, unsigned char *out, size_t *outlen, const unsigned char *in, size_t inlen);
 int EVP_PKEY_decrypt(EVP_PKEY_CTX *ctx, unsigned char *out, size_t *outlen, const unsigned char *in, size_t inlen);
 
+/* Abstraction of the EVP_MD_CTX struct */
+struct evp_md_ctx_st {
+    bool is_initialized;
+    EVP_PKEY *pkey;
+    size_t digest_size;
+};
```

Comment:
Could you group struct together right after definitions at the beginning of the file? Also, could you submit a PR to https://github.com/awslabs/aws-verification-model-for-libcrypto with these changes on the OpenSLL model?

Reply:
Done! And will submit a PR just as soon as I've made the other changes.